script/incremental_build.sh

```diff
@@ -5,13 +5,21 @@ readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly REPO_DIR="$(dirname "$SCRIPT_DIR")"
 
 source "$SCRIPT_DIR/common.sh"
+source "$SCRIPT_DIR/nnbd_plugins.sh"
 
 if [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ]; then
   PUB=pub.bat
 else
   PUB=pub
 fi
 
+# Plugins that are excluded from this task.
+ALL_EXCLUDED=("")
+# Exclude nnbd plugins from stable.
+if [[ "$CHANNEL" -eq "stable" ]]; then
+  ALL_EXCLUDED=($EXCLUDED_PLUGINS_FROM_STABLE)
```

Comment:
nit: can you print to the log here that we're skipping checks for the following plugins?

Reply:
Done.